ratpack-core/src/test/groovy/ratpack/http/client/HttpReverseProxySpec.groovy

```diff
@@ -420,8 +424,160 @@ connection: close
     status << noBodyResponseStatuses()
   }
 
+  def "can proxy a 100-continue request"() {
```

Comment:
Missing some coverage for redirect case IIUC. Would be worth also testing a double redirect case.

Reply:
Added specs for this.